src/components/route/booklist.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import '../css/bookStyle.css';
+
+const BookList = (prop) => {
+  const { book, deleteBook } = prop;
+  return (
+    <li key={book.id} className="book-list">
+      <h2>{book.title}</h2>
+      <p>
+        Author:
+        {book.author}
+      </p>
+      <button type="button" onClick={() => deleteBook(book.id)}>Remove</button>
+    </li>
```

Comment:
- [ ] While you did a fantastic job @misikir21 ,👍 as mentioned by the previous reviewer I'd like to kindly request that you reconsider adding the <Book> component to your project, and the BookList rendering a list of books using the <Book> component. Having a dedicated 'Book' component allows for better separation of concerns. Each component can focus on its specific task, making the codebase more organized and easier to maintain.😊

Reply:
@IngDavidTM  i made the requested change but for some reason, I am unable to push it this is the last chance I get to repeat this module if I am not approved today and the time is ticking to submit.The next restart date is 2024 I can't afford to wait until then.i